src/components/my-career/SkillsRecommendationCourses.jsx

```diff
@@ -0,0 +1,132 @@
+import React, {
+  useContext, useMemo, useState, useEffect,
+} from 'react';
+import { useHistory } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { AppContext } from '@edx/frontend-platform/react';
+import { CardGrid, Hyperlink } from '@edx/paragon';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import SearchCourseCard from '../search/SearchCourseCard';
+
+import { useDefaultSearchFilters, useSearchCatalogs } from '../search/data/hooks';
+import { UserSubsidyContext } from '../enterprise-user-subsidy';
+import { SubsidyRequestsContext } from '../enterprise-subsidy-requests';
+
+const SkillsRecommendationCourses = ({ index, subCategoryName, subCategorySkills }) => {
```

Comment:
I think it would be better to initialize `index` in this component instead of passing it in props from above. What do you say?